agent/exploits/cve_2018_14558.py

```diff
@@ -21,28 +21,30 @@
 TARGET_PATH = "/goform/setUsbUnload/.js?deviceName=A;"
 MAX_DELAY_DIFFERENCE = 5
 DELAYS = [30, 40, 50, 60]
+MAX_REDIRECTS = 2
 
 
 @exploits_registry.register
 class CVE201814558Exploit(definitions.Exploit):
     """CVE-2018-14558: Tenda AC7, AC9, and AC10 Routers Command Injection Vulnerability"""
 
-    def __init__(self) -> None:
-        self.session = requests.Session()
-
     def accept(self, target: definitions.Target) -> bool:
+        session = requests.Session()
+        session.max_redirects = MAX_REDIRECTS
         target_uri = f"{target.scheme}://{target.host}:{target.port}"
         try:
-            resp = self.session.get(target_uri, verify=False, timeout=DEFAULT_TIMEOUT)
+            resp = session.get(target_uri, verify=False, timeout=DEFAULT_TIMEOUT)
```

Comment:
Why is there 2 styles of specifying verify=False? once as an arg, once as an attribute  to the session instance ?